src/dune_engine/action_ext.ml

```diff
@@ -0,0 +1,52 @@
+open Import
```

Comment:
It seems to me that the intent o this file is to store type and module type definitions related to the [Action] module.

If that's indeed the case, why not just add these definitions to action_intf.ml which seem to serve the same purpose?

Reply:
The reason why it's separate is more for my own convenience. If I'm working on a feature, it pays off to keep as much code as possible in a separate file to minimize the opportunities for conflicts. I can move all this code to Action_intf and Action once we're more or less settled on the interface.